app/models/film.rb

```diff
@@ -11,7 +11,7 @@ def format_runtime
     s = "#{run_time}"
     if run_time > 60
       minutes = run_time % 60
-      hours = run_time / 60
+      hours = (run_time / 60).floor
       if hours < 2
```

Comment:
This is a case where Rails can really help you out. They have a [pluralize](https://api.rubyonrails.org/v4.2.11/classes/String.html#method-i-pluralize) method. It does exactly what you're trying to do here on it's own. You could replace this with something like "#{hours}#{"hr".pluralize(hours)}" and it would take care of some of the edge cases you're missing right now. For instance if a film was less than an hour your implementation would say "0hr" rather than "0hrs"

Reply:
Cool, that really simplifies things